app/parsers/pbcore_manifest_parser.rb

```diff
@@ -147,7 +147,7 @@ def set_objects
       asset_bulkrax_identifier =  if asset.bulkrax_identifier
                                     asset.bulkrax_identifier
                                   else
-                                    Bulkrax.fill_in_blank_source_identifiers.call(self, "Asset-#{index}-1")
+                                    Bulkrax.fill_in_blank_source_identifiers.call(self, "Asset", asset_id, 1)
```

Comment:
```suggestion
                                    Bulkrax.fill_in_blank_source_identifiers.call(self, "Asset", index, 1)
```

Reply:
we explicitly want asset_id here - we're done with marking these 1,2,3. we want the pbcore GUID instead so they can easily be traced back to the pbcore data. by using this externally consistant key, we can always find the entries and all items connected to a specific xml file.

Reply:
ok. I thought we were going with something more representative like the `id`, but each of the other calls to this method still use `index` as the value being passed in.

Reply:
the other calls were existing methods - I changed what was passed in to index in the method, but I can be more consistent about it. Let me push up more changes. I am leaving the actual lambda alone - since it shouldn't care what value is passed in there keeping it called `index` makes sense to me, even if we only send it the GUID for now. it has a separation of concerns.

Reply:
thanks for the clarification and explicit argument names!